charts/spark-operator-chart/templates/deployment.yaml

```diff
@@ -74,6 +74,11 @@ spec:
         args:
         - -v={{ .Values.logLevel }}
         - -logtostderr
+        - -max-queue-time-without-update-in-minutes=30
+        - -queue-cleaner-interval-in-minutes=10
```

Comment:
Can we make these two values configurable? Would there be a possibility for users to modify these values?

Reply:
+1